### ISSUE TITLE
cartodb.js build 3.15.8 broken

### ISSUE THREAD

**Issue by azamorano** [CartoDB/cartodb.js#1028]
The build is broken with the following output:
```
==> default: [31m>> [39mDestination not written because minified CSS was empty.
==> default: [4mRunning "imagemin:distCSSImages" (imagemin) task[24m
==> default: [31mFatal error: Cannot read property 'contents' of undefined[39m
```

```
Ran HOME=~ubuntu grunt build returned 3
```

**Comment by xavijam** [comment]
We should generate a new one. @alonsogarciapablo

**Comment by xavijam** [comment]
Fixed on new 3.15.9 version.
